modules/profile/README.md

```diff
@@ -0,0 +1,90 @@
+# SCC Profile Module
+
+This module creates an SCC Profile (https://cloud.ibm.com/docs/security-compliance?topic=security-compliance-build-custom-profiles&interface=ui). A profile is a grouping of controls that can be evaluated for compliance.
+
+The module supports the following actions:
+- Create SCC Profile
+
+### Usage
+
+```hcl
+provider "ibm" {
+  ibmcloud_api_key = "XXXXXXXXXX" # pragma: allowlist secret
+  region           = "us-south"
+}
+
+# - SCC Profile
+module "create_scc_profile" {
+  source           = "terraform-ibm-modules/scc/ibm//modules/profile"
+  instance_id = module.create_scc_instance.guid
+  controls = [
+    {
+      control_library_name    = "IBM Cloud Framework for Financial Services",
+      control_library_version = "1.6.0"
+      control_list = [
+        "AC",
+        "AC-1",
+        "AC-1(a)",
+      ]
+    },
+    {
+      control_library_name    = "CIS IBM Cloud Foundations Benchmark",
+      control_library_version = "1.0.0"
+      control_list = [
+        "1.16",
+        "1.18",
+        "1.19",
+        "1.4",
+      ]
+    },
```

Comment:
maybe add another control library example here which does not specify a `control_list` (in which case it adds all control right?)
Also add some code comments here to explain